src/main/clojure/clara/linter.cljc

```diff
@@ -0,0 +1,28 @@
+(ns clara.linter
+  (:require [clojure.string :as clj-str]))
```

Comment:
We seem a bit inconsistent with how we alias this ns in this project, but typically it's `string`:

```
clojure/clara/tools/inspect.cljc
21:    [clojure.string :as str])

clojure/clara/rules/engine.cljc
6:            [clojure.string :as string]

clojure/clara/rules/dsl.clj
6:            [clojure.string :as string]

clojure/clara/rules/compiler.clj
8:            [clojure.string :as string]
```